music/src/transcription/model.ts

```diff
@@ -87,12 +86,12 @@ export class OnsetsAndFrames {
                              tf.io.loadWeights(manifest, this.checkpointURL));
     this.build(vars);
     Object.keys(vars).map(name => vars[name].dispose());
+    this.initialized = true;
     if (warmup) {
-      tf.tidy(() => {
-        this.processBatches(tf.zeros([1, 16, MEL_SPEC_BINS]), 8, 8, 1);
-      });
+      const ctx = new OfflineAudioContext(1, SAMPLE_RATE, SAMPLE_RATE);
```

Comment:
For this to work in Safari do you need to do the weird `new (window.OfflineAudioContext || window.webkitOfflineAudioContext)` or no?